Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,863 @@
+{
```

Comment:
<p>Format: Should there be a newline after "remove spikes" perhaps?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>